src/functions/generatePlateSerialNumber.ts

```diff
@@ -0,0 +1,20 @@
+import {APIGatewayProxyResult, Context, Handler} from "aws-lambda";
+import {NumberService} from "../services/NumberService";
+import {HTTPResponse} from "../utils/HTTPResponse";
+import {PlateSerialNumber} from "../models/NumberModel";
+import {DynamoDBService} from "../services/DynamoDBService";
+
+const generatePlateSerialNumber: Handler = async (event: any, context?: Context): Promise<APIGatewayProxyResult> => {
+    const numberService = new NumberService(new DynamoDBService());
+
+    return numberService.createPlateSerialNumber(1, null)
```

Comment:
Something minor but we could probably flatten the promise here since createPlateSerialNumber seems to be returning a promise and we use async in the function block above.
So we could possibly remove the then() and just call await on createPlateSerialNumber then return the HTTPResponse. We could use a try catch block and in the catch bock we throw.
What do you think?